Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -3926,6 +4065,13 @@ public void AlerterPressed(bool pressed)
             TrainControlSystem.AlerterPressed(pressed);
         }
 
+        public enum TrainType { Pax, Cargo };
+        public TrainType SelectedTrainType = TrainType.Pax;
+        public void ChangeTrainTypePaxCargo()
+        {
+            SelectedTrainType = SelectedTrainType == TrainType.Pax ? SelectedTrainType = TrainType.Cargo : TrainType.Pax;
```

Comment:
Could you provide more information about this selector? Is it only used for cruise control (in real trains), or should it change also braking related parameters? I was planning to add something similar for brake operation.

Reply:
I may write large story about brakes. Because the brakes are still simulated same as back when we only have MSTS. Not a very large, challenging or breathtaking information. But this needs complete review (I mean the brakes) or small steps for us, until we are on the Moon, (or Mars nowadays).
The TrainType cargo : pax is now used for cruisecontrol only to select different time in reduced force, before the whole train is in tension on all couplers.

Reply:
Thanks, let's keep the button for cruise control only then.